apps/web/src/services/asset-inventory/store/collector-form-store.ts

```diff
@@ -1,6 +1,19 @@
 import { defineStore } from 'pinia';
 
-export const useCollectorFormStore = defineStore('collector-form', {
-    state: () => ({}),
-    actions: {},
+interface CollectorFormState {
+    selectedCollector: any; // TODO: CollectorPlugin Model
```

Comment:
That makes sense. how about just `collector`

Reply:
or `collectorInfo`?

Reply:
Is that for original data of collector? or updating data?
If it is updating data, I think it would be far better to set properties of collector as flat.
If it is original data, what about `originData` or `originCollector`?

Reply:
If I understand correctly, Original data is correct.
The `collector` stored in the store will be used for viewing and for creating or updating.